src/main/java/com/massivecraft/factions/cmd/CmdSethome.java

```diff
@@ -48,7 +48,7 @@ public void perform() {
         // Can the player set the home for this faction?
         // Check for ALLOW access as well before we check for role.
         if (faction == myFaction && access != Access.ALLOW) {
-            if (!Permission.SETHOME_ANY.has(sender) || !assertMinRole(Role.MODERATOR)) {
+            if (!Permission.SETHOME_ANY.has(sender, true) || !assertMinRole(Role.MODERATOR)) {
```

Comment:
ok good, that should fix the current issue we're having of not having a verbose failure reason. What about if they aren't a mod?

Reply:
If they aren't mod they will just get generic no perm (If the player doesn't have Allow)